app/views/api/services/show.html.slim

```diff
@@ -122,16 +122,18 @@ section class="service-widget"
           - if service.deployment_option.present?
             - deployment_option_phrased = t(service.proxy.deployment_option, scope: "deployment_options.phrased")
           - unless service.has_traffic?
-            li.item.integration
+            li class=("item integration #{'is-hidden' if apiap?}")
```

Comment:
we should remove it from the html if apiap, not hide it with css

Reply:
The paragraph is not rendered when apiap, check here
https://github.com/3scale/porta/pull/1342/files#diff-2eaca81d1f3c950ba0e8e12823649c5dR128

This class is to not show an empty `li`

Reply:
you're right, i was too quick, ignore!